step03/db.py

```diff
@@ -0,0 +1,68 @@
+from flask import g
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+import models
+from models import Profiles
+
+Base = models.Base
+
+engine = create_engine('sqlite+pysqlite:///imagely.db')
+Base.metadata.create_all(engine)
+Session = sessionmaker(bind=engine)
+session = Session()
+
+
+def registered_user(github_id):
```

Comment:
the convention is to name it something like `is_registered` or `is_registered_user`